packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -119,6 +119,18 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
   checkForNoWrap?: boolean;
 }
 
+export enum FocusZoneTabbableElements {
+
+  // allowTabKey is false
+  none = 0,
+
+  // allowTabKey is true
+  all = 1,
+
+  // allowTabKeyOnInput is true and allowTabKey is false
+  inputOnly = 2
+}
```

Comment:
If you update the prop name to something like `handleTabKey`  the enum name and string values should be updated... for the values maybe `never = 0, always= 1, inputOnly = 2`